include.js

```diff
@@ -2903,8 +2903,14 @@ let cards = [
     author: 'Kalliran',
     githubLink: 'https://github.com/Kalliran'
   },
-];
-
+  {
+    artName: 'Snow Animation', // change this to the name of your artwork
+    pageLink: './Art/imankitajamwal/index.html', // change this
+    imageLink: './Art/imankitajamwal/Snow_Globe.gif', // change this
+    author: 'Ankita', // use your name
+    githubLink: 'https://github.com/imankitajamwal' // change this
+  } 
+  ];
```

Comment:
# Invalid paths x folder contribution and comments
The paths do not match the folder name - you are using the right pattern but your folder is name differently:
Change the folder name to be named after your github username
and please remove the comments :)